staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -418,11 +421,11 @@ func ServerPreferredNamespacedResources(d DiscoveryInterface) ([]*metav1.APIReso
 	}), all), err
 }
 
-func (d *DiscoveryClient) clusterAwarePath(path string) string {
-	if d.cluster == "" {
-		return path
+func (d *DiscoveryClient) clusterAwarePath(p string) string {
```

Comment:
(No action required) Maybe avoid unnecessary var name brevity except for index vars and receivers?